metricbeat/docs/modules/apache.asciidoc

```diff
@@ -18,13 +18,13 @@ here (add link). Below is an example of a configuration option:
 [source,yaml]
 ----
 metricbeat.modules:
-  - module: apache
-    metricsets: ["status"]
-    enabled: true
-    period: 1s
+- module: apache
```

Comment:
This is directly generated from the config.yml file. It looks like you edited it here directly?